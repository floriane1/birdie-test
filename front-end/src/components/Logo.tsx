import styled from 'styled-components';

const Logo = styled.img`
  width: 200px;
  margin-top: 30px;
`;

export default Logo;