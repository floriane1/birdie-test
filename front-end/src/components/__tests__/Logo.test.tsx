import * as React from 'react';
import { shallow, configure } from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import Logo from '@App/components/Logo';

const LogoUrl = require('@App/components/app/logo.svg');

configure({ adapter: new Adapter() });

describe('Logo', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Logo src={LogoUrl} />);
    expect(wrapper).toMatchSnapshot();
  });
});
