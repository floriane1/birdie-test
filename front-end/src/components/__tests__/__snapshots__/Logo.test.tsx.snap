// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
      width: 200px;
      margin-top: 30px;
    ",
        ],
      },
      "displayName": "styled.img",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "img",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
  src="logo.svg"
/>
`;
